src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -695,6 +711,52 @@ private void determineUnitsToAttackWith(final List<ProTerritory> prioritizedTerr
       Map<Unit, Set<Territory>> sortedUnitAttackOptions =
           tryToAttackTerritories(prioritizedTerritories, alreadyMovedUnits);
 
+      // Clear bombers
+      for (final Territory t : attackMap.keySet()) {
+        attackMap.get(t).getBombers().clear();
+      }
+
+      // Get all units that have already moved
+      final Set<Unit> alreadyAttackedWithUnits = new HashSet<>();
+      for (final Territory t : attackMap.keySet()) {
+        alreadyAttackedWithUnits.addAll(attackMap.get(t).getUnits());
+        alreadyAttackedWithUnits.addAll(attackMap.get(t).getAmphibAttackMap().keySet());
+      }
+
+      // Check to see if any territories can be bombed
+      final Map<Unit, Set<Territory>> bomberMoveMap = territoryManager.getAttackOptions().getBomberMoveMap();
+      for (final Unit unit : bomberMoveMap.keySet()) {
+        if (alreadyAttackedWithUnits.contains(unit)) {
+          continue;
+        }
+        Optional<Territory> maxBombingTerritory = Optional.empty();
+        int maxBombingScore = 3;
```

Comment:
Tracking the max bombing score across the loop so needs to be before the loop. Added constant for the value.